reference-artifacts/SAMPLE_CONFIGS/config.example.json

```diff
@@ -1252,6 +1259,50 @@
           "rsyslog-enforce-imdsv2": true,
           "rsyslog-root-volume-size": 100,
           "rsyslog-max-instance-age": 7
+        },
+        "opensearch-siem": {
+          "deploy": true,
+          "vpc-name": "Central",
+          "region": "${HOME_REGION}",          
+          "security-groups": [
+            {
+              "name": "opensearch-siem",
+              "inbound-rules": [
+                {
+                  "description": "Allow Traffic Inbound",
+                  "tcp-ports": [443],                  
+                  "source": "${RANGE-RESTRICT}"
+                }
+              ],
+              "outbound-rules": [
+                {
+                  "description": "All Outbound",
+                  "type": ["ALL"],
+                  "source": ["0.0.0.0/0"]
+                }
+              ]
+            }
+          ],
+          "app-subnets": [
+            {
+              "name": "App",
+              "az": "a"
+            },
+            {
+              "name": "App",
+              "az": "b"
+            }
+          ],          
+          "lambda-log-processing-role": "ASEA-OpenSearch-ProcessingRole",
```

Comment:
Change ASEA to ${ACCELERATOR_PREFIX_ND}.  Question - where is this role created?  Why is it created differently than the below role?

Reply:
The opensearch-instane-role was removed (config file now updated to reflect that).